Cloudinary.Test/AsyncMethodsTest.cs

```diff
@@ -249,6 +278,84 @@ public void TestUpdateTransformStrictAsync()
             Assert.AreEqual(false, getResult.Strict);
         }
 
+        [Test]
+        public void TestUpdateTransformWithExtensionAsync()
```

Comment:
try to unify these tests to one test with different inputs and expected results

Reply:
I merged the two I've added to 'TestUpdateTransformWithImplicitExtension'.